.circleci/config.yml

```diff
@@ -224,7 +224,7 @@ jobs:
   build-macos-cmake:
     macos:
       xcode: 14.3.1
-    resource_class: macos.m1.medium.gen1
+    resource_class: macos.m1.large.gen1
```

Comment:
Sadly we don't currently have access to this resource class - that's why we went for M1 mediums when we did the migration. You can see the policy evaluation failure on the CircleCI dashboard.